### PR TITLE
Major Updates 

### PR DESCRIPTION
**Pull request by hacker7599**
** Updated Main Site Content
** Added Contact Us Inquiry Feature.
** Added Sales inquiry in Admin
** Rectified Previous normalization Issues
** Updated Web routes in admin for sales 
** Updated Web routes for main site

### CODE REVIEW

File: app/Http/Controllers/MainSiteController.php [hunk 1]
```diff
@@ -2,13 +2,26 @@
...
     }
 
+    public function contactUs(ContactUsRequest $request)
+    {
+        $contact = new ContactUs();
+        $contact->name = $request->txtname;
+        $contact->email = $request->txtemail;
+        $contact->message = $request->txtmessage;
+        $contact->save();
+        return view('main.home');
```
Comment (jaydeepukani) [inline app/Http/Controllers/MainSiteController.php:22]: Don't do it like this. just redirect to main.home route.

File: app/Http/Requests/Main/ContactUsRequest.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'txtname' => ['required'],
+            'txtemail' => ['required'],
```
Comment (jaydeepukani) [inline app/Http/Requests/Main/ContactUsRequest.php:28]: Add proper validation like validating the email field, check the name is a string. make limit for store chars like min, max. Also, add validation to check message length.

File: app/Http/Controllers/MainSiteController.php (hunk 1 above)
Comment (hacker7599) [reply app/Http/Controllers/MainSiteController.php:22]: Done Resolved

File: app/Http/Requests/Main/ContactUsRequest.php (hunk 2 above)
Comment (hacker7599) [reply app/Http/Requests/Main/ContactUsRequest.php:28]: Done Resolved

**Review by jaydeepukani — APPROVED** [review]

**Pull request merged**